net/ngtcp2/Portfile

```diff
@@ -34,6 +30,12 @@ configure.args-append   --disable-silent-rules \
                         --enable-werror=no \
                         --with-gnutls=yes
 
+variant openssl description "Enable OpenSSL" {
+    depends_lib-append  port:openssl3
+    configure.args-append \
+                        --with-openssl=yes
```

Comment:
Just to make sure, can openssl and gnutls be used here simultaneously?

Reply:
Yes. Except those cases stated explicitly in the readme file (https://github.com/ngtcp2/ngtcp2?tab=readme-ov-file#crypto-helper-library), crypto library backends could coexist and are only leveraged when linked specifically by a program.

Reply:
Ok, do what you consider useful, I will test it on my end. And thank you for improving the port!